### PR TITLE
Remove unnecessary feeds from NuGet.config

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -7,16 +7,21 @@
...
+
     <add key="roslyn-analyzers" value="https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json" />
-    <add key="roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
-    <add key="vs-devcore" value="https://myget.org/F/vs-devcore/api/v3/index.json" />
+
+    <!-- Microsoft.VisualStudio.* packages (e.g. Microsoft.VisualStudio.ImageCatalog): https://github.com/dotnet/roslyn/issues/43242 -->
     <add key="vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
+
+    <!-- Microsoft.VisualStudio.ProjectSystem.Managed -->
+    <add key="roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```
Comment (jasonmalinowski) [inline NuGet.config:23]: Any idea why this is in the Roslyn feed versus...something else?

**Comment by tmat** [comment]
@jaredpar FYI

File: NuGet.config (hunk 1 above)
Comment (tmat) [reply NuGet.config:23]: Historical reasons I guess. It should be VSSDK or vs-impl feed.

Comment (tmat) [reply NuGet.config:23]: Filed https://github.com/dotnet/project-system/issues/6079

**Pull request merged**